default/stringtables/en.txt

```diff
@@ -2770,6 +2773,29 @@ Create Binary Save Files
 OPTIONS_USE_XML_ZLIB_SERIALIZATION
 Use Compression for XML Save Files 
 
+OPTIONS_CREATE_PERSISTENT_CONFIG
+Write Persistent Config File
+
+OPTIONS_CREATE_PERSISTENT_CONFIG_TOOLTIP_TITLE
+Create Persistent Config (Advanced Feature)
+
+OPTIONS_CREATE_PERSISTENT_CONFIG_TOOLTIP_DESC
+'''Uses your current config settings as persistent default settings.
```

Comment:
Avoid referring to the player/user in the second person. Maybe "Saves the current config settings as..."